rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.75.0"
+components = ["clippy", "rustfmt"]
```

Comment:
`rust-src` is probably necessary here to for jump-to-definition in rust-analyzer